contracts/ApeSwapMultiSwapRouter.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity =0.7.6;
+pragma abicoder v2;
+
+import './interfaces/IApeSwapMultiSwapRouter.sol';
+import './base/FactoryWhitelist.sol';
+import './V2SwapRouter.sol';
+import './V3SwapRouter.sol';
+import './base/MulticallExtended.sol';
+
+/// @title Uniswap V2 and V3 Swap Router
+contract ApeSwapMultiSwapRouter is IApeSwapMultiSwapRouter, V2SwapRouter, V3SwapRouter, MulticallExtended {
+    constructor(
+        address[] memory _factories,
+        address _WETH9
+    ) FactoryWhitelist(_factories) PeripheryImmutableState(address(0), _WETH9) {}
```

Comment:
Shouldn't be an issue. It's never used as everything has a factory/router as param. I would say leave as address(0).
Don't want to bother updating the package either as it's a uniswap package (might even make sense to use theirs instead of ours)

Reply:
I agree with using their package 👌